src/components/structures/RoomSubList.js

```diff
@@ -347,8 +347,14 @@ var RoomSubList = React.createClass({
         var label = this.props.collapsed ? null : this.props.label;
 
         let content;
-        if (this.state.sortedList.length === 0 && !this.props.searchFilter && this.props.extraTiles.length === 0) {
-            content = this.props.emptyContent;
+        if (this.state.sortedList.length === 0) {
```

Comment:
Should this not include `&& this.props.extraTiles.length === 0` ?

Reply:
Yup you're right, mine worked by an assumption that the current uses of `<RoomSubList />` are the only possible use cases but in future, we may have alternate ones.